components/infobox/commons/infobox_map.lua

```diff
@@ -35,6 +35,8 @@ function Map:createInfobox(frame)
 		Cell{name = 'Creator', content = {
 				args.creator or args['created-by'], args.creator2 or args['created-by2']}, options = { makeLink = true }
 		},
+		Cell{name = 'Release Date', content = {args.releasedate}},
+		Cell{name = 'Game', content = {args.game}, options = { makeLink = true }},
```

Comment:
see https://github.com/Liquipedia/Lua-Modules/pull/883#discussion_r775307064

Reply:
Ok, let's drop `game` from commons.

Reply:
Let's keep game/gameversion in custom for now at least then.

Reply:
Noted. Removing code for that